content/engineering/processes/definition-of-ready-done.md

```diff
@@ -1,46 +0,0 @@
----
-title: "Definition of Ready/Done"
-date: 2021-08-31T18:11:15+02:00
-draft: false
-description: "Definition of Ready and Done for each stage in the workflow"
-weight: 2
```

Comment:
@CodeNameNinja Why was this removed? I like this section :D Just the miro board was linking to the incorrect board.